lib/screens/SiteDetailScreen.dart

```diff
@@ -89,7 +89,7 @@ class _SiteDetailScreenState extends State<SiteDetailScreen> {
         }),
         refreshController: refreshController,
         onRefresh: () async {
-          if (site.connected) {
+          if (site.connected && site.status == "Connected") {
```

Comment:
nit: Maybe a constant? Meh.

Reply:
Oh man, yes. But also more!

`site.connected` really means `site.active` so I wanna rename that
`site.connected` should then become a boolean meaning "i am actually truly connected now" or replaced with an int status to check against, the computer counterpart to `site.status`
and finally `site.status` stays as the human string.